plugins/UiConfig/languages/pt-br.json

```diff
@@ -30,4 +30,15 @@
 	"Save settings": "Salvar configurações",
 	"Some changed settings requires restart": "Algumas configurações alteradas requrem reinicialização",
 	"Restart ZeroNet client": "Reiniciar cliente da ZeroNet"
```

Comment:
You missed a comma at the end of this line. Can you add it please?

Reply:
done

Reply:
You still haven't fixed the issue. There should be a comma (,) at the end of the line.

Reply:
now I made the correction